.changeset/chatty-crabs-tap.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/repo-tools': minor
+---
+
+Adds a lint rule to `repo schema openapi lint` to enforce `allowReserved: true`.
```

Comment:
For people who use these repo tools, they'd need an explanation of what the outcome could be in their repos after this and how to go about addressing it, right?

Reply:
Good call, I'll update.